ndk-sys/CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Unreleased
 
+# 0.4.2 (2022-11-23)
+
+- Regenerate against NDK 25.1.8937393. (#370)
+- Regenerate against NDK 25.0.8775105. (#324)
```

Comment:
In other repos (`ash`) I just fold these together, perhaps this should just be:

```suggestion
- Regenerate against NDK 25.1.8937393 with `bindgen 0.63`. (#324, #370)
```

(and imply a bindgen update)

as nobody cares about an intermediate update that wasn't even released.